test/integration/sf.e2e.ts

```diff
@@ -62,7 +71,7 @@ describe('Salesforce CLI (sf)', () => {
      * GLOBAL FLAGS
      *   <global flags>
      */
-    const regex = /^[A-Z].*\n\nUSAGE[\S\s]*\n\nFLAGS[\S\s]*\n\nGLOBAL FLAGS[\S\s]*$/g
+    const regex = /^.*?USAGE.*?FLAGS.*?GLOBAL FLAGS.*?(?!DESCRIPTION).*?(?!EXAMPLES).*?(?!FLAG DESCRIPTIONS).*?(?!CONFIGURATION VARIABLES).*?(?!ENVIRONMENT VARIABLES).*$/gs
```

Comment:
I extended this regex to confirm that the other section present when using `--help` are not present on `-h`